### PR TITLE
CS01 Interpolation using ML

### PR DESCRIPTION
**Pull request by Sameem25**
The following problem has been done using Normal Equation for faster calculation. Cost function so calculated finally is least.

### CODE REVIEW

File: pset0/CS01_interpolation.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import matplotlib.pyplot as plt
+get_ipython().magic('matplotlib inline')
+
+
+# In[35]:
+
+
+#user input values for 'Year' and the corresponding 'Profit'
+
+year_arr=[]
```
Comment (RudraNilBasu) [inline pset0/CS01_interpolation.py]: Space before and after =

File: pset0/CS01_interpolation.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+profit_arr=[]
```
Comment (RudraNilBasu) [inline pset0/CS01_interpolation.py]: Here too

**Review by RudraNilBasu — CHANGES_REQUESTED** [review]
* Make sure to send PRs from a new branch (other than master)
* Follow Bytes Club's commit message style
* Run the code through PyLint for coding guidelines

**Comment by Sameem25** [comment]
So will I create a new PR with from a new branch and correcting the errors?

**Comment by RudraNilBasu** [comment]
@Sameem25 keep it for now, create a new branch for next PRs

**Comment by 0xTheProDev** [comment]
@Sameem25 Along with modifying the current errors, make your solution to the point and modular. And the file should be inside directory `Sameem25`.

File: pset0/CS01_interpolation.py (hunk 2 above)
Comment (Sameem25) [reply pset0/CS01_interpolation.py]: Okay

**Comment by Sameem25** [comment]
Okay

**Comment by Sameem25** [comment]
Should I remove the graphs and make it short or reduce the number of code lines?

**Comment by 0xTheProDev** [comment]
@Sameem25 I think that's what I wanted! 👍

**Pull request closed without merging**